DATA/EQUIPMENT/commodity_good.ini

```diff
@@ -2210,3 +2210,18 @@ bad_sell_price = 0.9
 good_buy_price = 0.9
 shop_archetype = Equipment\models\commodities\nn_icons\cwire_rawmats_2.3db
 item_icon = equipment\models\icons\goods\Icon_Surprise_FLSR.3db
+
+;; named boxes
+
+[Good]
+nickname = named_box_king
+equipment = named_box_king
+category = commodity
+price = 50000000
```

Comment:
This high price also influences Death Penalty. Maybe it is better to lower the price for the item itself, and increase the BUY PRICE FACTOR in the market_misc.ini to 10 or something?